fairlearn/metrics/_extra_metrics.py

```diff
@@ -1,4 +1,4 @@
-# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Random question: why remove 'all rights reserved' ?

Reply:
Qouting our lawyers: “we no longer use ‘all rights reserved’”

Reply:
Fair enough :-)